src/frontend/js/widgets/SyllabusCourseRunsList/components/CourseProductItem/index.tsx

```diff
@@ -42,15 +43,21 @@ const CourseProductItem = ({ productId, courseCode }: Props) => {
   const ordersQuery = useOrders({
     product: productId,
     course: courseCode,
-    state: [OrderState.VALIDATED, OrderState.PENDING],
+    state: [OrderState.VALIDATED, OrderState.PENDING, OrderState.SUBMITTED],
   });
 
-  const order = useMemo(
-    () => ordersQuery.items.find(({ state }) => state === OrderState.VALIDATED),
-    [ordersQuery.items],
-  );
+  const order = useMemo(() => {
+    if (ordersQuery.items.length === 0) {
+      return undefined;
+    }
+    return ordersQuery.items.reduce((last, newOrder) =>
+      last.created_on > newOrder.created_on ? last : newOrder,
+    );
```

Comment:
I thought that our backend cannot have more that one order by `courseCode-productId`.